package.json

```diff
@@ -60,11 +60,10 @@
   "dependencies": {
     "@tensorflow/tfjs": "^1.5.1",
     "long": "^4.0.0",
```

Comment:
I don't remember what long is.  Is it related to a type that we can remove due to Protobuf?

Reply:
This comes with protobuf. It needs it to handle int64 type that's not natively supported in JS.